packages/datatrak-web/src/features/Survey/SurveyContext.tsx

```diff
@@ -21,20 +21,66 @@ const convertNumberToLetter = (number: number) => {
 };
 
 type SurveyFormContextType = {
-  formData: { [key: string]: any };
-  setFormData: (data: { [key: string]: any }) => void;
+  formData: Record<string, any>;
   activeScreen: SurveyScreenComponent[];
   isLast: boolean;
   numberOfScreens: number;
   screenNumber: number;
   screenHeader?: string;
   displayQuestions: SurveyScreenComponent[];
+  sideMenuOpen?: boolean;
+  surveyScreenComponents?: Record<number, SurveyScreenComponent[]>;
 };
 
-const SurveyFormContext = createContext({} as SurveyFormContextType);
+const defaultContext = {
+  formData: {},
+  activeScreen: [],
+  isLast: false,
+  numberOfScreens: 0,
+  screenNumber: 1,
+  screenHeader: '',
+  displayQuestions: [],
+  sideMenuOpen: false,
```

Comment:
I would have expected the sideMenu to default to open? It seems that way in the design

Reply:
When we went through the design with Flic the other day she said it defaulted to closed

Reply:
ah ok - in that case ignore what I said :)